### PR TITLE
Update rai component to 0.6.0

### PR DESCRIPTION
**Pull request by tongyu-microsoft**

### CODE REVIEW

**Review by RubyZ10 — APPROVED** [review]

**Review by vinuthakaranth — DISMISSED** [review]

File: responsibleai/tabular/components/src/rai_component_utilities.py [hunk 1]
```diff
@@ -130,22 +147,52 @@ def load_mltable(mltable_path: str) -> pd.DataFrame:
         tbl = mltable.load(mltable_path)
         df: pd.DataFrame = tbl.to_pandas_dataframe()
     except Exception as e:
-        _logger.info("Failed to load MLTable")
-        _logger.info(e)
+        _logger.info(f"Failed to load {mltable_path} as MLTable. ")
+        raise e
```
Comment (imatiach-msft) [inline responsibleai/tabular/components/src/rai_component_utilities.py:151]: are we sure we should raise here?  The other changes look like whitespace but this one seems more serious.

**Review by gaugup — DISMISSED** [review]

File: responsibleai/tabular/components/src/rai_component_utilities.py [hunk 2]
```diff
@@ -130,22 +147,52 @@ def load_mltable(mltable_path: str) -> pd.DataFrame:
...
 
 
 def load_parquet(parquet_path: str) -> pd.DataFrame:
-    _logger.info("Loading parquet file: {0}".format(parquet_path))
-    df = pd.read_parquet(parquet_path)
+    _logger.info(f"Attempting to load {parquet_path} as parquet dataset")
+    try:
+        df = pd.read_parquet(parquet_path)
+    except Exception as e:
+        _logger.info(f"Failed to load {mltable_path} as MLTable. ")
```
Comment (imatiach-msft) [inline responsibleai/tabular/components/src/rai_component_utilities.py:160]: same thing here and below, this looks like a functional change

**Review by imatiach-msft — DISMISSED** [review]

File: responsibleai/tabular/components/src/rai_component_utilities.py (hunk 1 above)
Comment (tongyu-microsoft) [reply responsibleai/tabular/components/src/rai_component_utilities.py:151]: this is from Kin's PR https://github.com/Azure/RAI-vNext-Preview/pull/162/files

File: responsibleai/tabular/components/src/rai_component_utilities.py (hunk 2 above)
Comment (tongyu-microsoft) [reply responsibleai/tabular/components/src/rai_component_utilities.py:160]: yes, we want to copy/paste everything we changed from vNext repo to azureml-assets. So we can keep azureml-assets with latest RAI component updates

**Review by imatiach-msft — APPROVED** [review]

**Pull request merged**